### DISCUSSION TITLE
prunenotifications was removed, what is the replacement?

### DISCUSSION THREAD

**Discussion by saschabrockel** [healthchecks/healthchecks#1082] [Q&A] [answered]
Hey I had a monthly script that called `manage.py prunenotifications` but not I realized that `prunenotifications` was removed. There is something called `pruneobjects` but I cannot find documentation about it. Is it the replacement?

**Comment by cuu508** [comment, marked as answer]
Notifications are now being cleaned up automatically during pinging:

https://github.com/healthchecks/healthchecks/blob/e048ec4c487643cc2c4eef48eab1790c151c58cb/hc/api/models.py#L537

`pruneobjects` removes objects from [external object storage](https://github.com/healthchecks/healthchecks?tab=readme-ov-file#external-object-storage). It is mentioned in the "Database Cleanup" section in README and [also here](https://healthchecks.io/docs/self_hosted/#database-cleanup).
